aif360/datasets/standard_dataset.py

```diff
@@ -118,7 +118,7 @@ def __init__(self, df, label_name, favorable_classes,
                 unprivileged_values = list(set(df[attr]).difference(vals))
             else:
                 # find all instances which match any of the attribute values
-                priv = np.logical_or.reduce(np.equal.outer(vals, df[attr]))
+                priv = np.logical_or.reduce(np.equal.outer(vals, df[attr])).to_numpy()
```

Comment:
this should be `...df[attr].to_numpy()))` actually

Reply:
Oh, yeah, too rush... Thanks, @hoffmansc.